### PR TITLE
don't load stylesheet module in certain pages

### LINKED ISSUE #2474: Disable stylesheet module on sensitive pages
Whole module or per row?

And show a notification with a long cooldown (1 week?) of "subreddit stylesheets are disabled here, go to settings to enable them"

Sensitive pages are /about/moderators, account preferences, anything lese?

### PR DESCRIPTION
**Pull request by andytuba**
also includes some interesting changes to .pageType that hopefully won't break everything

closes #2474

### CODE REVIEW

**Comment by mc10** [comment]
+1 this is cool.

File: lib/core/utils.js [hunk 1]
```diff
@@ -386,26 +388,13 @@ RESUtils.pageType = function() {
...
-	} else if (RESUtils.regexes.prefs.test(currURL)) {
-		pageType = 'prefs';
-	} else if (RESUtils.regexes.stylesheet.test(currURL)) {
-		pageType = 'stylesheet';
-	} else {
-		pageType = 'linklist';
-	}
+	var defaultPageType = 'linklist';
+	var pageTypes = [ 'wiki', 'search', 'profile', 'comments', 'inbox', 'submit', 'prefs', 'stylesheet' ];
+	var unsortedPageTypes = Object.getOwnPropertyNames(RESUtils.regexes).filter(function(pageType) { return pageType !== 'all' && pageTypes.indexOf(pageType) === -1; }).reverse();
```
Comment (erikdesjardins) [inline lib/core/utils.js]: This breaks everything because most pages that used to be `linklist` match `subredditPostListing`

Comment (andytuba) [inline lib/core/utils.js]: Dang it, I meant to take that part out before I merged. I'll fix it in the morning, thanks for calling it out.

Comment (erikdesjardins) [inline lib/core/utils.js]: Just in case: don't forget to add `subredditAbout` and `account` to `pageTypes`

Comment (andytuba) [inline lib/core/utils.js]: Sorry about that. Thanks for the reminder.  341a3c9

**Pull request merged**